zarr/tests/test_storage.py

```diff
@@ -798,7 +799,7 @@ class TestDBMStoreNDBM(TestDBMStore):
 
             def create_store(self):
                 path = tempfile.mktemp(suffix='.ndbm')
-                atexit.register(os.remove, path)
+                atexit_rmglob(path + '*')
```

Comment:
These are still suppose to be passed to `atexit.register`, correct?

Reply:
Yes, that happens within atexit_rmglob.

Reply:
Doh no it doesn't.